src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ selectedTabId, onTabSelected, tabs }) => {
+  const fondTab = tabs.find(tab => tab.id === selectedTabId.id)
```

Comment:
The find function returns the object not a boolean. The current implementation will not correctly identify the selected tab. Instead, it will always return the first tab id as the 'selectedTabId' because every non-null object is truthy in JavaScript.